docs/kubespray_gcp.md

```diff
@@ -0,0 +1,49 @@
+## Deploy Kubernetes with Kubespray
+## Example on Google Cloud Platform
```

Comment:
one more `#` here

Reply:
this this not `baremetal` config?
We also talk to add link to - https://github.com/kubernetes/ingress-nginx/blob/master/docs/deploy/baremetal.md or this is not needed?
Maybe another section?

Reply:
If you were thinking about nginx used in our Ingress then baremetal is described in [Ingress subchart readme](https://github.com/IntelAI/inference-model-manager/blob/master/helm-deployment/ing-subchart/README.md)